config/aaa.py

```diff
@@ -189,3 +248,249 @@ def delete(address):
     config_db.connect()
     config_db.set_entry('TACPLUS_SERVER', address, None)
 tacacs.add_command(delete)
+
+
+@click.group()
+def radius():
+    """RADIUS server configuration"""
+    pass
+
+
+@click.group()
+@click.pass_context
+def default(ctx):
+    """set its default configuration"""
+    ctx.obj = 'default'
+radius.add_command(default)
+
+
+@click.command()
+@click.argument('second', metavar='<time_second>', type=click.IntRange(1, 60), required=False)
+@click.pass_context
+def timeout(ctx, second):
+    """Specify RADIUS server global timeout <1 - 60>"""
```

Comment:
The command is "config radius timeout".
Please correct the comments

Reply:
The comment string is the help string in the CLI output. I've followed the same style as TACACS+. Which comment should I correct?

`admin@sonic:~$ sudo config  radius timeout --help
Usage: config radius timeout [OPTIONS] <time_second>

  Specify RADIUS server global timeout <1 - 60>

Options:
  -h, -?, --help  Show this message and exit.
admin@sonic:~$`